templates/nginx.conf

```diff
@@ -4,14 +4,32 @@ upstream pretalx_{{ pretalx_instance_identifier }}_server {
 
 proxy_cache_path /tmp/nginx-pretalx-{{ pretalx_instance_identifier }} levels=1:2 keys_zone=pretalx_static_{{ pretalx_instance_identifier }}:10m inactive=60m max_size=250m;
 
+{% if pretalx_tls %}
 server {
+    listen 80;
+    listen [::]:80;
+
+    server_name {{ pretalx_domain }}{% if pretalx_alternate_domains %} {{ pretalx_alternate_domains }}{% endif %};
+    return 301 https://{{ pretalx_domain }}$request_uri;
+}
+{% endif %}
+
+server {
+    {% if pretalx_tls %}
     listen 443 ssl;
     listen [::]:443 ssl;
+    {% else %}
```

Comment:
Hmm, I'm not sure about this. pretalx should not run without HTTPS, and if people want to run it without, I'd recommend they use their own nginx configuration. I don't really want to make it easier to do than it has to be.